e2e-tests/cypress/e2e/journeys/insurance/your-business/broker/validation/broker-postcode-validation.spec.js

```diff
@@ -0,0 +1,216 @@
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = '22 2AA';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has 2 digits in the second part', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = 'SW1 22A';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has 3 digits in the second part', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = 'SW1 222';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has all letters in the second part', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = 'SW1 AAA';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has more than 7 characters', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = 'SW1A 2AAA';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has less than 5 characters', () => {
+    it('should display validation errors', () => {
+      const field = broker[FIELD_ID];
+
+      field.yesRadioInput().click();
+
+      const {
+        errorField, errorIndex, expectedErrorsCount, errorMessageFormat,
+      } = ERROR_ASSERTIONS;
+
+      const inputValue = 'S 2AA';
+
+      cy.submitAndAssertFieldErrors(errorField, inputValue, errorIndex, expectedErrorsCount, errorMessageFormat);
+    });
+  });
+
+  describe('when the postcode has a space', () => {
```

Comment:
Can we wrap the last 2 tests in a `describe('valid postcode ...` so it's clearer?